graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/FunctionBodyGenerator.java

```diff
@@ -155,7 +159,7 @@ public FunctionBodyNode generateInitFunctionBody(AuthConfig authConfig) {
         // Generate {@code self.apiKeyConfig = apiKeyConfig.cloneReadOnly();} assignment node
         AssignmentStatementNode apiKeyConfigAssignmentStatementNode = generateApiKeyConfigAssignmentStatementNode();
 
-        assignmentNodes.add(httpConfigNode);
+        assignmentNodes.addAll(httpClientConfigNode);
```

Comment:
Let's handle the generic errors and return as a `ballerina/graphql` module specific error.